--- origsrc/marble-4.12.5/CMakeLists.txt	2014-03-23 02:31:52.000000000 -0500
+++ src/marble-4.12.5/CMakeLists.txt	2014-05-01 10:18:00.441623700 -0500
@@ -27,6 +27,7 @@ include( MarbleMacros )
 # if this option is set, srtm.jpg will not be installed but the generated tiles instead
 option(QTONLY "Create Marble version without KDE dependencies" OFF)
 option(MOBILE "Create a Marble version optimized for handheld devices")
+option(TOUCH  "Create a Marble version optimized for Meego devices" OFF)
 
 ####################################################
 # Specific options for if building with Qt or kde4 libs
--- origsrc/marble-4.12.5/src/apps/CMakeLists.txt	2014-03-23 02:31:52.000000000 -0500
+++ src/marble-4.12.5/src/apps/CMakeLists.txt	2014-05-01 10:18:41.152793400 -0500
@@ -6,6 +6,8 @@ if (NOT QTONLY)
   add_subdirectory(marble-kde)
 endif()
 
+if( TOUCH )
 if( QT_QTDECLARATIVE_FOUND OR Qt5Quick_FOUND )
     add_subdirectory(marble-touch)
 endif( QT_QTDECLARATIVE_FOUND OR Qt5Quick_FOUND )
+endif( TOUCH )
