--- origsrc/kdeedu-4.5.5/marble/src/plugins/render/aprs/posix_qextserialport.cpp	2010-06-08 05:10:44.000000000 -0500
+++ src/kdeedu-4.5.5/marble/src/plugins/render/aprs/posix_qextserialport.cpp	2011-01-14 03:27:42.227670300 -0600
@@ -12,6 +12,10 @@
 #include <QMutexLocker>
 #include <QDebug>
 
+#ifdef Q_OS_CYGWIN
+#define FIONREAD TIOCINQ
+#endif
+
 void QextSerialPort::platformSpecificInit()
 {
     fd = 0;
@@ -918,12 +922,16 @@ unsigned long QextSerialPort::lineStatus
         if (Temp&TIOCM_RTS) {
             Status|=LS_RTS;
         }
+#ifdef TIOCM_ST
         if (Temp&TIOCM_ST) {
             Status|=LS_ST;
         }
+#endif
+#ifdef TIOCM_SR
         if (Temp&TIOCM_SR) {
             Status|=LS_SR;
         }
+#endif
     }
     return Status;
 }
